src/renderers/art/ReactARTFiber.js

```diff
@@ -485,6 +485,10 @@ const ARTRenderer = ReactFiberReconciler({
     // Noop
   },
 
+  getChildHostContext() {
+    return null;
+  },
+
```

Comment:
We should make these optional so that they don't have to be added to every renderer. That way we can also do things like:

```js
const { getChildHostContext } = config;
if (config.getChildHostContext) {
  var data = getSomeData();
  return config.getChildHostContext(data);
}
```

That way if the compiler knows that it is always undefined, it can dead-code eliminate everything in the `if` statement.